src/hotspot/share/nmt/memReporter.cpp

```diff
@@ -594,7 +605,7 @@ void MemSummaryDiffReporter::print_virtual_memory_diff(size_t current_reserved,
 }
 
 
-void MemSummaryDiffReporter::diff_summary_of_type(MemTag mem_tag,
+void MemSummaryDiffReporter::diff_summary_of_tag(MemTag mem_tag,
```

Comment:
Move out into mainline PR

Reply:
Fixed in commit "missed flag/tag -> type" (7640108).